README.md

```diff
@@ -17,11 +17,42 @@ Analyze, Annotate, Publish. Turn documents into data.
 2. Enter the directory - `cd documentcloud`
 3. Run the dotenv initialization script - `python initialize_dotenvs.py`
    This will create files with the environment variables needed to run the development environment.
-4. Start the docker images - `inv up`
-   This will build and start all of the docker images using docker-compose. It will bind to port 80 on localhost, so you must not have anything else running on port 80. The invoke tasks specify the `local.yml` configuration file for docker-compose. If you would like to run docker-compose commands directly, set the environment variable `export COMPOSE_FILE=local.yml`.
+4. Start the docker images - `export COMPOSE_FILE=local.yml; docker-compose up`
+    -   This will build and start all of the docker images using docker-compose. It will bind to port 80 on localhost, so you must not have anything else running on port 80. The `local.yml` configuration file has the docker-compose details.
 5. Set `api.dev.documentcloud.org` and `minio.documentcloud.org` to point to localhost - `sudo echo "127.0.0.1 api.dev.documentcloud.org minio.documentcloud.org" >> /etc/hosts`
 6. Enter `api.dev.documentcloud.org/` into your browser - you should see the Django API root page.
-7. Install and run [Squarelet](https://github.com/muckrock/squarelet) and the [DocumentCloud frontend](https://github.com/muckrock/documentcloud-frontend) following the instructions in their repos to view the full-stack application.
+7. Install and run [Squarelet](https://github.com/muckrock/squarelet) and the [DocumentCloud frontend](https://github.com/muckrock/documentcloud-frontend) following the instructions in their repos to view the full-stack application. 
+    - `inv createsuperuser` in Squarelet (not DocumentCloud).
```

Comment:
I believe his should be `inv manage creaturesuperuser`

Reply:
I believe this is resolved on the latest commit.

Reply:
Actually, we need to revert this to what was in the last version:
```
stall ✗)]$ /Users/thadk/Library/Python/3.8/bin/inv manage createsuperuser
Container squarelet_squarelet_postgres_1  Running
Container squarelet_squarelet_mailhog_1  Running
Container squarelet_squarelet_nginx_1  Running
PostgreSQL is available
Superuser creation skipped due to not running in a TTY. You can run `manage.py createsuperuser` in your project to create one manually.
```